clients/spring-boot-starter-camunda-sdk/src/test/java/io/camunda/spring/client/config/CredentialsProviderSelfManagedTest.java

```diff
@@ -92,10 +93,13 @@ void shouldHaveZeebeAuth() throws IOException {
                             .put("token_type", "bearer")
                             .put("expires_in", 300))));
 
+    // when
     credentialsProvider.applyCredentials(headers::put);
-    credentialsProvider.applyCredentials(headers::put);
+
+    // then
     assertThat(credentialsProvider).isExactlyInstanceOf(OAuthCredentialsProvider.class);
-    assertThat(headers).containsEntry("Authorization", "Bearer " + accessToken);
+    final var authorizationValue = headers.get("Authorization");
+    assertThat(authorizationValue).isEqualTo("Bearer " + accessToken);
```

Comment:
🙃 The map `containsEntry` would also work fine I guess.

Reply:
👍 I'll revert this change